### PR TITLE
Update OWNERS file

### PR DESCRIPTION
**Pull request by awgreene**
Removes @Bowenislandsongas and a duplicate mention of @benluddy from the list of reviewers.

### CODE REVIEW

**Comment by kevinrizza** [comment]
While we're here, can you also add @timflannagan1 to the list of reviewers?

File: OWNERS [hunk 1]
```diff
@@ -17,10 +17,9 @@ reviewers:
   - anik120
   - exdx
   - awgreene
-  - Bowenislandsong
-  - benluddy
   - hasbro17
   - ankitathomas
   - joelanford
+  - @timflannagan1
```
[the remark this answers is not in the document]
Comment (awgreene) [reply OWNERS:23]: Removed the `@`.

[the remark this answers is not in the document]
Comment (timflannagan) [reply OWNERS:23]: I also removed the trailing `1` from handle recently lol

[the remark this answers is not in the document]
Comment (timflannagan) [reply OWNERS:23]: (I'm almost positive prow will complain about this)

Comment (timflannagan) [inline OWNERS:23]:
```suggestion
  - timflannagan
```

**Review by dinhxuanvu — APPROVED** [review]
/lgtm

**Comment by kevinrizza** [comment]
Force merging so we don't keep wasting CI cycles on flaky tests

**Pull request merged**